packages/sources/moonbeam-address-list/test/integration/setup.ts

```diff
@@ -0,0 +1,12 @@
+import { Adapter } from '@chainlink/external-adapter-framework/adapter'
+import { customSettings } from '../../src/config'
+import { addressEndpoint } from '../../src/endpoint/address'
+
+export const createAdapter = (): Adapter<typeof customSettings> => {
```

Comment:
I don't like this pattern that's been added to all EAs, but we can change it later, just a note